### PR TITLE
Implement supported inverter brands on inverter page

### LINKED ISSUE #252: Implement supported inverter brands on inverter page
## Description

We'd like to display the icons for all the inverters that are supported by Enode! On mobile, this will look like a lil scrolling section, and on desktop, this will be in a modal. 

Screen Shot 2023-04-26 at 12 01 40 AM
Screen Shot 2023-04-26 at 12 02 00 AM
Screen Shot 2023-04-26 at 12 02 17 AM

## Todos

These points are rough guidelines. **Please** feel free to discuss the best way to design this component with others on the team!

- [x] grab brand logos—these hopefully are publically available somewhere, @AndrewLester or @ericcccsliu might do this for you
- [x] implement mobile inverter display
- [x] implement desktop inverter display modal 
- [x] put "what's this" display in a modal for mobile 

## Useful links

- y'all know where the figma is by now 

## Time estimate
tbd

### PR DESCRIPTION
**Pull request by neha-vard**
## Description
Display the icons for all the inverters that are supported by Enode. On mobile, this will look like a lil scrolling section, and on desktop, this will be in a modal.

Fixes #252

## Todos
Improve styling
- better align inverter brand names/logos
- center info modal

## Screenshots
image
image
image

### CODE REVIEW

File: lib/components/form/LinkInverters.tsx [hunk 1]
```diff
@@ -55,17 +63,52 @@ const LinkInverters: FC<{ siteUUID: string }> = ({ siteUUID }) => {
...
+            <img src={growatt.src} alt="ema"></img>
+            <div className="my-2 self-center text-ocf-gray-300">Growatt</div>
+          </div>
+          <div className="flex flex-row">
+            <img src={solaredge.src} alt="ema"></img>
+            <div className="my-2 self-center text-ocf-gray-300">SolarEdge</div>
+          </div>
+          <div className="flex flex-row">
+            <img src={solis.src} alt="ema"></img>
+            <div className="my-2 self-center text-ocf-gray-300">Solis</div>
```
Comment (ericcccsliu) [inline lib/components/form/LinkInverters.tsx]: We could maybe make this code simpler by using the same idea from the help pages—some object mapping brands to srcs?

Comment (neha-vard) [reply lib/components/form/LinkInverters.tsx]: Yes, I was planning on doing that actually!

File: lib/components/form/LinkInverters.tsx [hunk 2]
```diff
@@ -2,13 +2,49 @@ import Link from 'next/link';
...
 import { useRouter } from 'next/router';
 import { fetcher } from '~/lib/swr';
 import { useIsMobile } from '~/lib/utils';
+import ema from '../../../public/inverters/ema.png';
+import enphase from '../../../public/inverters/enphase.png';
+import fronius from '../../../public/inverters/fronius.png';
+import goodwe from '../../../public/inverters/goodwe.png';
+import growatt from '../../../public/inverters/growatt.png';
+import solaredge from '../../../public/inverters/solaredge.png';
+import solis from '../../../public/inverters/solis.png';
```
Comment (ericcccsliu) [inline lib/components/form/LinkInverters.tsx]:
```suggestion
import ema from '~/public/inverters/ema.png';
import enphase from '~/public/inverters/enphase.png';
import fronius from '~/public/inverters/fronius.png';
import goodwe from '~/public/inverters/goodwe.png';
import growatt from '~/public/inverters/growatt.png';
import solaredge from '~/public/inverters/solaredge.png';
import solis from '~/public/inverters/solis.png';
```

File: lib/components/form/LinkInverters.tsx [hunk 3]
```diff
@@ -2,13 +2,49 @@ import Link from 'next/link';
...
+
+const SupportedInverters = () => {
+  return (
+    <div>
+      {Object.keys(brands).map((brand) => {
+        return (
+          <div key={brand} className="flex flex-row">
+            <img
+              src={brands[brand as keyof typeof brands].src}
+              alt={brand}
```
Comment (ericcccsliu) [inline lib/components/form/LinkInverters.tsx]:
```suggestion
                alt={`${brand} logo`}
```

**Review by ericcccsliu — APPROVED** [review]
LGTM

**Pull request merged**